Added functionality to delete selected row/column

Yagotshus99 opened the pull request:
Added a delete row and delete column button to delete selected row/column.

In this patch, a user can select a row or column and then press the "delete selected row" or "delete selected column" button to delete the selected row/column. The buttons also work if a user selects a specific cell instead of a whole row/column. The first row and first column cannot be deleted currently. 

Testing Steps

1. Run main.py to start the application.
2. Select a column (not the first) and press the delete column button, make sure column is deleted.
3. Select the first column, press the delete button, and make sure it is NOT deleted.
4. Select a row (not the first) and press the delete row button make sure row is deleted
5. Select the first row and press the delete row button, make sure row is NOT deleted.
6. Select a row and click the delete column button, make sure nothing is deleted.
7. Select a column and click the delete row button, make sure nothing is deleted.

Type: New Feature

Yagotshus99 committed:
Added functionality to delete selected row/column

Testing Steps

1. Run main.py to start the application.
2. Select a column (not the first) and press the delete column button, make sure column is delete.
3. Selected the first column and press the delete button and make sure it is NOT deleted.
4. Select a row (not the first) and press the delete row button make sure row is deleted
5. Select the first row and press the delete row button, make sure row is NOT deleted.
6. Select a row and click the delete column button, make sure nothing is deleted.
7. Select a column and click the delete row button, make sure nothing is deleted.

Yagotshus99 committed:
Merge branch 'main' into dev_table_delete_row_delete_col_buttons

Yagotshus99 committed:
Fixed alignment issues in table panel

Jcobo314 reviewed (approved):
Code is formatted correctly and the testing worked perfectly. Good to merge

Yagotshus99 committed:
Bug fix

Fixed not being able to delete first row. Added logic to determine whether a row or column was selected.

Yagotshus99 committed:
Merge branch 'main' into dev_table_delete_row_delete_col_buttons

Yagotshus99 committed:
removed debugging print statement
